optimum/habana/transformers/integrations/finegrained_fp8.py

```diff
@@ -0,0 +1,312 @@
+import os
+from enum import Enum
+from typing import Callable, Optional, Tuple
+
+import torch
+import torch.nn as nn
+from torch.nn import functional as F
+from transformers.utils import (
+    is_accelerate_available,
+)
+
+from optimum.utils import logging
+
+
+if is_accelerate_available():
+    from accelerate import init_empty_weights
```

Comment:
```suggestion
from accelerate import init_empty_weights

from optimum.utils import logging   
```

In OH accelerate is always available. No additional checks are required.

Reply:
Done